client/web/src/cody/team/TeamMemberList.test.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, expect, it } from 'vitest'
+import { formatInviteDate } from './TeamMemberList'
+
+// This is a bit weird test, but I wanted to give the design some examples for how the formatting looks,
+// and this was the easiest way to do that. Rather than throwing away the test, I'm leaving it here.
+describe('getFilterFnsFromCodyContextFilters', () => {
+    it('looks good for typical input', async () => {
+        // These are in the format of the normal output of our Go back end
```

Comment:
"in the format of the normal output of our Go back end" is a bit too wishy washy. There is a precise format that we expect, and so we should call that out.

Read: broadly speaking, our REST API types from the backend represent all dates as ISO-8601 strings, specifically using the RFC 3339 profile. (Which you are parsing in the `formatInviteDate` function.)

- https://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats
- https://go.dev/src/time/format.go
- https://en.wikipedia.org/wiki/ISO_8601

```suggestion
        // The backend returns times as ISO-8601 (RFC3339) strings.
```